diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -316,21 +359,17 @@ function StudentQuiz(runtime, element, initData) {
       } else {
         if (currentIndex > newIndex) {
           // allow to move backwards without validate & save
+          common.publishEvent({
+            event_type: 'xblock.diagnostic_feedback.quiz.question.reloading',
+            reloading_question: newIndex
+          });
           return true;
         }
         var selectedChoice = $(visibleAnswerChoice, element).find(selectedStudentChoice).val();
 
         if (selectedChoice != "" && selectedChoice != undefined) {
```

Comment:
@jareerahsan You didn't touch this line directly but it would be worth updating this to use `!==` instead of `!=`. Alternatively, if you just want to check if the value of `selectedChoice` is falsy, you could also shorten it to `if (selectedChoice) {`.